EMaligner/EMaligner.py

```diff
@@ -342,27 +300,26 @@ def assemble_from_hdf5(self, filename, zvals, read_data=True):
                             indptr,
                             f.get('indptr')[()][1:] + indptr[-1])
                     weights = np.append(weights, f.get('weights')[()])
+
+                    k = 0
+                    while True:
+                        name = 'rhs_%d' % k
+                        if name in f.keys():
```

Comment:
right now, up to 2 entries.
for now, I'll keep this, as I plan to come back to the hdf5 file interface format once the dust settles.
there is an idea in my mind of extending rhs to > 2 entries, so that we can take advantage of a large matrix factorization and loop relatively quickly through multiple regularizations without re-doing the entire factorization. But, I haven't worked that out completely yet.